src/Sylius/Bundle/ProductBundle/Behat/ProductContext.php

```diff
@@ -291,4 +291,32 @@ public function theFollowingOptionTranslationsExist(TableNode $table)
 
         $manager->flush();
     }
+
+    /**
+     * @Given /^There is (enabled|disabled) product named "([^""]*)"$/
+     */
+    public function thereIsProduct($enabled, $name)
+    {
+        $product = $this->getRepository('product')->findOneBy(array('name' => $name));
+
+        $isEnabled = false;
+        if ('enabled' == $enabled) {
```

Comment:
Due to PHP dynamic typing they are always useful :) In this case not so much, but it's better to get used to them to avoid nasty bugs.